staging_service/import_specifications/file_writers.py

```diff
@@ -87,24 +89,22 @@ def _check_import_specification(types: dict[str, dict[str, list[Any]]]):
         # replace this with jsonschema? don't worry about it for now
         _check_string(datatype, "A data type")
         spec = types[datatype]
-        if type(spec) != dict:
+        if type(spec) != dict:  # noqa: E721
```

Comment:
presumably flake8 wants you to use `isinstance` here? I'd leave it without the `noqa` and fix it in one of the upcoming PRs.